inst/shiny/server.R

```diff
@@ -141,6 +141,9 @@ shinyServer(function(input, output, session) {
     yearRange <- unique(w.use$YEAR)
     w.use.sub <- subset_wuse(w.use, data.elements, area.column, areas)
     
+    if(!length(unique(w.use.sub$YEAR))==length(w.use.sub$YEAR)) {
+      stop("Recieved multiple areas for the same year, this is not currently supported. Maybe you selected a whole state? Please select a list of unique areas.")}
+    
```

Comment:
Adding USSTATEALPHACODE is great for the plotting, but breaks the ranking because of this issue. See issue #64 for discussion.